### PR TITLE
Build script for bazel deb

### PR DESCRIPTION
**Pull request by trainman419**
Simple build script to invoke bazel's deb build, and include the zoox bazel wrapper in the bazel repo

### CODE REVIEW

File: scripts/BUILD [hunk 1]
```diff
@@ -44,3 +44,11 @@ filegroup(
     ],
     visibility = ["//:__pkg__"],
 )
+
+exports_files(["zsh_completion/_bazel"])
+
+sh_binary(
+    name = "bazel_run",
+    srcs = ["bazel-run.sh"],
```
Comment (vincer) [inline scripts/BUILD]: Where is bazel-run.sh?

Comment (trainman419) [inline scripts/BUILD]: In the scripts directory: https://github.com/zooxco/bazel/blob/master/scripts/bazel-run.sh

Comment (vincer) [inline scripts/BUILD]: Ah, missed that somehow.

File: deb.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```
Comment (marczych) [inline deb.sh]: No `set -euo pipefail`?

File: scripts/BUILD [hunk 3]
```diff
@@ -44,3 +44,11 @@ filegroup(
     ],
     visibility = ["//:__pkg__"],
 )
+
```
Comment (vincer) [inline scripts/BUILD]: A comment here to split off Zoox additions like you did in bazel.sh would be nice. Will make it easier if/when we merge upstream changes.

File: deb.sh (hunk 2 above)
Comment (trainman419) [inline deb.sh]: Fixed

File: scripts/BUILD (hunk 3 above)
Comment (trainman419) [inline scripts/BUILD]: Done.

**Pull request merged**